### PR TITLE
jump to code

### PR DESCRIPTION
**Pull request by pahans**

### CODE REVIEW

File: modules/web/js/ballerina/components/statement-decorator.jsx [hunk 1]
```diff
@@ -58,6 +60,15 @@ class StatementDecorator extends React.Component {
...
         this.props.model.remove();
+    }
+
+	onJumptoCodeLine() {
+			const {expression} = this.props;
+			const {renderingContext: {ballerinaFileEditor}} = this.context;
+
+			const container = ballerinaFileEditor._container;
+			$(container).find('.view-source-btn').trigger('click');
+			ballerinaFileEditor.getSourceView().jumpToLine({expression});
```
Comment (hemikak) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: Code formatting issue? issue in indents.

Comment (nadeeshaan) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: Indentation need to be fixed

File: modules/web/js/ballerina/components/statement-decorator.jsx [hunk 2]
```diff
@@ -42,10 +42,12 @@ class StatementDecorator extends React.Component {
         dragDropManager.on('drag-start', this.startDropZones.bind(this));
         dragDropManager.on('drag-stop', this.stopDragZones.bind(this));
 
-		this.state = {innerDropZoneActivated: false,
-                      innerDropZoneDropNotAllowed: false,
-                      innerDropZoneExist: false,
-                      showActions: false };
+		this.state = {
```
Comment (maheeka) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: indentation issue?

File: modules/web/js/ballerina/components/statement-decorator.jsx (hunk 1 above)
Comment (pahans) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: fixed with https://github.com/ballerinalang/composer/pull/2011

File: modules/web/js/ballerina/components/statement-decorator.jsx (hunk 2 above)
Comment (pahans) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: fixed with https://github.com/ballerinalang/composer/pull/2011

File: modules/web/js/ballerina/components/statement-decorator.jsx (hunk 1 above)
Comment (pahans) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: fixed with https://github.com/ballerinalang/composer/pull/2012

Comment (pahans) [inline modules/web/js/ballerina/components/statement-decorator.jsx]: fixed with https://github.com/ballerinalang/composer/pull/2012

**Pull request merged**